packages/esm-appointments-app/src/appointments/forms/create-edit-form/appointments-form.component.tsx

```diff
@@ -252,15 +254,13 @@ const AppointmentForm: React.FC<AppointmentFormProps> = ({ appointment, patientU
           </TimePicker>
 
           <TimePicker
-            disabled={!appointmentService}
```

Comment:
No idea what this was doing before, but since you removed it in 3 places, can you describe the issue?

Reply:
This was disabling setting the time until you selected a service, which didn't seem to make sense to me, as you could pick a *date* (and fill out any of the other fields) before selecting a service.  (And it was not at all clear that was why the inputs were disabled).

You still can't submit until you select a service.